src/trace_data.rs

```diff
@@ -174,3 +175,8 @@ impl ExecutionResources {
         true
     }
 }
+
+#[derive(Clone, Default, Debug, Serialize, Deserialize)]
+pub struct OnchainData {
+    pub l2_l1_message_sizes: Vec<usize>,
+}
```

Comment:
Remember to reuse this one in forge after https://github.com/foundry-rs/starknet-foundry/pull/1766 is merged